### ISSUE TITLE
Compiling for slow machines OpenGL 1.3

### ISSUE THREAD

**Issue by mills32** [sm64pc/sm64pc#295]
Great port!.

I want to compile the game using opengl 1.3 for a slow PC, but I get this error:
**undefined`` reference to `gfx_opengl_api`**

Thanks!

**Comment by coltongit** [comment]
Can you show the error during the compile process? It might be a simple fix from the sounds of this. Will test a bit later to see if I can reproduce your issue

**Comment by mills32** [comment]
The error shows at the end, just wen it's linking all the .o
`-march=native -Llib -lpthread -lglew32 `sdl2-config --static-libs -lm -lglu32 -lsetupapi -ldinput8 -luser32 -lgdi32 -limm32 -lole32 -loleaut32 -lshell32 -lwinmm -lversion -luuid -lopengl32 -static -no-pie
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: build/us_pc/src/pc/pc_main.o:pc_main.c:(.rdata$.refptr.gfx_opengl_api[.refptr.gfx_opengl_api]+0x0): undefined reference to `gfx_opengl_api'
collect2.exe: error: ld returned 1 exit status
make: *** [Makefile:928: build/us_pc/sm64.us.f3dex2e.exe] Error 1`

**Comment by coltongit** [comment]
@fgsfdsfgs looks like you might need to fix your renderer again

**Comment by mills32** [comment]
Can you explain how to do it? I'm not a very skilled programmer. If it is not easy, I just can live without it. :)

Thanks again.

**Comment by AaronMT** [comment]
@fgsfdsfgs semi related, how often will this repo pull and rebase off n64decomp for upstream fixes?

**Comment by mills32** [comment]
Ups, sorry I was compilling the nighty, The master did not show errors with the opengl legacy enabled.

**Comment by fgsfdsfgs** [comment]
LEGACY_GL isn't even *in* master. I would still appreciate if you gave me the log.

**Comment by mills32** [comment]
Sorry, maybe is something unrelated to your sources and that script has a bug.

Just in case here you have:

-**build.sh**: the script someone created
[build.sh.txt](https://github.com/sm64pc/sm64pc/files/4726306/build.sh.txt)

-**log_master.txt**: choosing "master" and LEGACY_GL, no error.
[log_master.txt](https://github.com/sm64pc/sm64pc/files/4726310/log_master.txt)

-**log_nightly.txt**: choosing "nightly"  and LEGACY_GL. It does not create the exe, (log does not show the last part about undefined reference to 'gfx_opengl_api' I posted above).
[log_nightly.txt](https://github.com/sm64pc/sm64pc/files/4726313/log_nightly.txt)


I compiled for 32bit, 60fps and LEGACY_GL.

I have to say, the legacy opengl works great on that "slow" PC, (an intel atom at 1.5GHz, with the worst intel HD you can imagine).

Thanks.

**Comment by mills32** [comment]
I'm going to forget about that script and compile as shown in the wiki, that build script keeps showing random errors.

**Comment by fgsfdsfgs** [comment]
Yeah that does seem to have compiled correctly. One tell tale sign of the legacy GL renderer working is that it spews a bunch of information into the console on startup, if you run it from one. You should probably try the regular renderer since apparently that was working fine for you when you were building master.
Don't know why you were getting that error using the script
